x/payment/keeper/msg_server_test.go

```diff
@@ -0,0 +1,6 @@
+package keeper_test
+
+// func setupMsgServer(t testing.TB) (types.MsgServer, context.Context) {
+// 	k, ctx := keepertest.PaymentKeeper(t)
+// 	return keeper.NewMsgServerImpl(*k), sdk.WrapSDKContext(ctx)
+// }
```

Comment:
Same q as for `x/payment/genesis_test.go`

Reply:
I actually wanted to include this, along with some test utils, but I think it will have to be done in another PR, as this one is already too crowded. 

deleted for now https://github.com/celestiaorg/lazyledger-app/pull/127/commits/0d0b88a989b182b3876cbb501291a9b6f798f2d1